acceptance/setup/000-delete-puppet-when-none.rb

```diff
@@ -0,0 +1,4 @@
+require 'beaker-puppet/helpers/setup/common'
+extend Beaker::DSL::Helpers::Setup::Common
+
+delete_puppet_when_none
```

Comment:
Is there a way to push these steps into beaker-puppet so that we can update/change the presuite without having to touch all of the component repos?

Reply:
I don't know, I was having a difficult way figuring out how best to do this. The testing format requires each test to be kicked off from seperate files. I'd love to pair with you on this to figure it out if we can